oshmem/include/shmem.h.in

```diff
@@ -48,7 +48,6 @@
 #endif
 
 #include <shmem-compat.h>
-#include <shmemx.h>
```

Comment:
:+1:
This change is necessary for compliance with the spec.  On page 5, lines 6-7 it says:  "The OpenSHMEM header files for C and Fortran must contain only the interfaces and constant names defined in this specification."